### PR TITLE
[MVI-887] Logging Refactor

### PR DESCRIPTION
**Pull request by avitaltune**
new PR for logging refactor.

### CODE REVIEW

**Comment by avitaltune** [comment]
sample 
`{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "cost.heyzap: Starting"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "Run: Starting"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "Worker: Start"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "cost.heyzap", "version": "69dddd5b70.9", "message": "Validate Client Credentials"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "Client Credentials: Valid"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "cost.heyzap", "version": "69dddd5b70.9", "message": "Validate Alpha Credentials"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "tune_mv_integration", "version": "1.21.15", "message": "Alpha Credentials: Valid"}
{"asctime": "2017-09-07 16:38:46 +0300", "levelname": "INFO", "name": "cost.heyzap", "version": "69dddd5b70.9", "message": "Begin"}
{"asctime": "2017-09-07 16:38:47 +0300", "levelname": "INFO", "name": "requests_mv_integrations", "version": "00.05.03", "message": "Campaigns Stats: Request: Finished", "request_time_msecs": 686}
{"asctime": "2017-09-07 16:38:47 +0300", "levelname": "INFO", "name": "cost.heyzap", "version": "69dddd5b70.9", "message": "Get Campaigns", "campaign count": 1}
{"asctime": "2017-09-07 16:38:48 +0300", "levelname": "INFO", "name": "requests_mv_integrations", "version": "00.05.03", "message": "Revenue: Request: Finished", "request_time_msecs": 854}
{"asctime": "2017-09-07 16:38:48 +0300", "levelname": "INFO", "name": "cost.heyzap", "version": "69dddd5b70.9", "message": "Get Revenue Stats", "App count": 2}
{"asctime": "2017-09-07 16:38:49 +0300", "levelname": "INFO", "name": "requests_mv_integrations", "version": "00.05.03", "message": "Revenue: Request: Finished", "request_time_msecs": 858}`

**Comment by zaquestion** [comment]
A+++++++ leaving it to @ohad for approval

File: logging_mv_integrations/tune_logging.py [hunk 1]
```diff
@@ -1,358 +1,38 @@
...
-        """
-        # print('TuneLogging', 'create_logger')
-        logging._checkLevel(self.logger_level)
+    handler = logging.StreamHandler()
+    handler.setFormatter(formatter)
 
-        if self.logger_level == logging.NOTSET:
-            self.logger_level = logging.INFO
+    logger = logging.getLogger(logger_name)
+    logger.setLevel(logging.INFO)
```
Comment (ohadtune) [inline logging_mv_integrations/tune_logging.py]: We should set the `logger_level `. and a safety check. like
`if logger_level == logging.NOTSET:		
    logger_level = logging.INFO`

File: logging_mv_integrations/tune_logging.py [hunk 2]
```diff
@@ -1,358 +1,38 @@
...
-        """Generate Logger
 
-        Args:
-            integration_name:
-            logger_name:
-            logger_level:
+def get_logger(logger_name, logger_version=None, logger_format=None, logger_level=None):
 
-        Returns:
+    formatter = LoggingJsonFormatter(
```
Comment (ohadtune) [inline logging_mv_integrations/tune_logging.py]: for backward compatibility need to check  `logger_format`.
if its STANDARD setting simple formatter using just `fmt`

File: logging_mv_integrations/tune_logging.py [hunk 3]
```diff
@@ -6,7 +6,9 @@
 class CustomAdapter(logging.LoggerAdapter):
 
     def process(self, msg, kwargs):
-        if 'extra' in kwargs and kwargs['extra'] is not None:
```
Comment (avitaltune) [inline logging_mv_integrations/tune_logging.py]: addressing @hasIan's comment

Comment (hasIan) [reply logging_mv_integrations/tune_logging.py]: 🙌

**Pull request merged**